src/components-examples/cdk-experimental/popover-edit/index.ts

```diff
@@ -45,6 +45,7 @@ const EXAMPLES = [
   ],
   declarations: EXAMPLES,
   exports: EXAMPLES,
+  entryComponents: EXAMPLES,
```

Comment:
Wouldn't this do the _opposite_ of what we want under ViewEngine? (always include the code even if it's not used)

Reply:
My understanding is that this tells NGC to generate factories for thr components. They are lazily loaded when needed. Previously, all factories were referenced, so all examples loaded. It should be an improvement for VE too.